src/popup/router/components/TransactionOverview.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+  <div class="transaction-overview">
+    <div class="title">
+      {{ transaction.title }}
+      <span class="description">{{ transaction.desc }}</span>
+    </div>
+
+    <div class="parties">
+      <Avatar v-bind="transaction.sender" />
+      <div class="mid">
+        <TriangleRight />
+        <div class="line" />
+      </div>
+      <div v-if="transaction.aens" class="avatar">
+        <AensIcon class="icon" />
+      </div>
+      <div v-else-if="transaction.contractCreate" class="avatar">
+        <ContractIcon class="icon" />
+      </div>
+      <Avatar v-else v-bind="transaction.recipient" />
+    </div>
+
+    <div class="details">
+      <div class="sender">
+        <a
+          :href="connect ? app.host : getExplorerPath(transaction.sender.address)"
+          target="blank"
+          class="name"
+        >
+          <TruncateMid :str="transaction.sender.name || transaction.sender.label" />
+        </a>
+        <CopyAddress :value="transaction.sender.address" />
+      </div>
+      <div class="recipient">
+        <span
+          v-if="transaction.aens || transaction.contractCreate"
+          :class="{ aens: transaction.aens }"
+          class="name"
+        >
+          {{ transaction.recipient.label }}
+        </span>
+        <a
+          v-else
+          :href="getExplorerPath(transaction.recipient.address)"
+          target="blank"
+          class="name"
+        >
+          <TruncateMid :str="transaction.recipient.name || transaction.recipient.label" />
+        </a>
+        <CopyAddress v-if="transaction.recipient.address" :value="transaction.recipient.address" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import { OBJECT_ID_TX_TYPE } from '@aeternity/aepp-sdk/es/tx/builder/schema';
+import { TX_TYPES, TX_TYPES_MDW } from '../../utils/constants';
+import TriangleRight from '../../../icons/triangle-right.svg?vue-component';
+import ContractIcon from '../../../icons/contract-create.svg?vue-component';
+import AensIcon from '../../../icons/aens.svg?vue-component';
+import Avatar from './Avatar';
+import TruncateMid from './TruncateMid';
+import CopyAddress from './CopyAddress';
+
+export default {
+  name: 'TransactionOverview',
+  components: {
+    Avatar,
+    TruncateMid,
+    CopyAddress,
+    TriangleRight,
+    AensIcon,
+    ContractIcon,
+  },
+  props: {
+    tx: { type: Object },
+    claim: { type: Boolean },
+    event: { type: String },
+    contract: { type: String },
+    connect: { type: Boolean },
+    app: { type: Object },
```

Comment:
please specify for each property the default value or mark it as required, otherwise, linter settings from #877 will warn about this